_explore/economic-impact/gdp.html

```diff
@@ -113,7 +113,7 @@ <h1 data-toggler="filters" data-filter-description="" class="filter-description"
 
         <h3 class="region-header-category">GDP by state</h3>
         <table class="subregions">
-          <thead class="region-header"></thead>
+          <tbody class="region-header"></tbody>
```

Comment:
This is an unfortunate change, as this is `thead`. If anyone has any suggestions for how to get a thead to play nice and be 100% width of its parent div, please let me know and I will correct this.